metaci/testresults/tests/conftest.py

```diff
@@ -0,0 +1 @@
+from metaci.fixtures.data_fixtures import *
```

Comment:
Why not just make the data_fixtures globally available from the top-level conftest.py? (That'll probably require moving the factories to a separate module to avoid import cycles.)

Reply:
I had played around with that first, but could never get it to work. Maybe because of the factories being in the top-level conftest.py?